docs/source/api/notebooks/Notebook_3_Model_Variants.ipynb

```diff
@@ -0,0 +1,379 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Notebook 3: Use GraphStorm APIs for Reproducing Built-in Models Variants\n",
+    "\n",
+    "This notebook demonstrates how to use GraphStorm's APIs to reproduce various GraphStorm built-in GNN models such as RGAT, and HGT, for different tasks.\n",
+    "\n",
+    "In this notebook, we use different ``GSgnnEncoder`` modules, and set the corresponding arguments in a GNN model, hence reproducing several GraphStorm built-in GNN models, such as `RGAT`, and `HGT`. Using the same pipelines demonstratred in the **Notebook 1** and **Notebook 2**, users can easily conduct node classification and link prediction task on the ACM dataset created by the **Notebook 0: Data Prepare**. \n",
+    "\n",
+    "### Prerequsites\n",
+    "\n",
+    "- GraphStorm installed using pip. Please find [more details on installation of GraphStorm](https://graphstorm.readthedocs.io/en/latest/install/env-setup.html#setup-graphstorm-with-pip-packages).\n",
```

Comment:
```suggestion
    "- GraphStorm is installed using pip. Please find [more details on installation of GraphStorm](https://graphstorm.readthedocs.io/en/latest/install/env-setup.html#setup-graphstorm-with-pip-packages).\n",
```

Reply:
Removed the `installed using pip` in all notebooks